lib/index.js

```diff
@@ -11,22 +10,24 @@ module.exports = function () {
   console.log('\n' + chalk.underline.blue('Yeoman Doctor'));
   console.log('Running sanity checks on your system\n');
 
-  eachAsync(objectValues(rules), (rule, i, cb) => {
-    rule.verify(err => {
-      console.log((err ? symbols.error : symbols.success) + ' ' + rule.description);
+  (async () => {
+    await Promise.all(
+      objectValues(rules).map(rule => {
```

Comment:
Can we make this into `Object.values()` now?

Reply:
@SBoudrias Sure. Code updated.